Add NODE_ENV configuration for production cyf-hosting deployments (#1727)
CodeYourFuture/curriculum: org-cyf-guides/content/deploying/cyf/backend/index.md

![Configure git branch](customise-branch.png)

## Add NODE_ENV to Your Project

> **Required for:** All JavaScript/Node.js projects (frontend and backend)

Reviewer: ```suggestion
**Required for:** All Node.js projects (anything that has a `package.json` file)
```
